<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCalendario</class>
 <widget class="QDialog" name="DialogCalendario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#DialogCalendario{
background-color:qlineargradient(spread:pad, x1:0.058, y1:0.136364, x2:1, y2:1, stop:0 rgba(0, 255, 127, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>1041</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255);
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string>GESTIONE DEGLI APPUNTAMENTI</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>421</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nuovoappbtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuovo Appuntamento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminaappbtn">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina Appuntamento</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="AppuntamentiTable">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>671</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Codice</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stato</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Esito</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="ricercaappCombo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="showdate_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>610</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Ricerca per</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chiudiappbtn">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Chiudi Appuntamento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeBtn">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>670</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>img/home.png</normaloff>img/home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1271</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>img/farmaciaa.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>calendarWidget</zorder>
  <zorder>nuovoappbtn</zorder>
  <zorder>eliminaappbtn</zorder>
  <zorder>AppuntamentiTable</zorder>
  <zorder>ricercaappCombo</zorder>
  <zorder>showdate_2</zorder>
  <zorder>chiudiappbtn</zorder>
  <zorder>homeBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
